tools/project_manager/autorelease.inc

```diff
@@ -126,27 +126,26 @@ function autorelease_for_round($round)
         //     An SQL condition that defines the kind of project
         //     that can be released by this queue.
         //
-        // $qd->release_criterion
-        //     A PHP condition that specifies the circumstances under which
+        // $qd->projects_target
+        // $qd->pages_target
+        //     Integers that specify the circumstances under which
         //     this queue can release a project.
-        //     It can reference 'projects' and 'pages', which denote (respectively)
-        //      -- the number of projects matching this queue's project_selector
-        //         that are currently available in the round, and
-        //      -- their total number of pages in any state other than 'saved'.
+        //
+        // For more details, see SETUP/site_admin_notes.txt under "Defining queues".
 
         $qd->cooked_project_selector = cook_project_selector($qd->project_selector);
 
         // Release available projects of a given type (defined by $qd->project_selector)
-        // as long as the $qd->release_criterion is true for projects of that type.
+        // as long as the condition implied by the targets is true for projects of that type.
 
         echo "\n";
         echo "Considering {$qd->name} projects...\n";
         echo "\n";
         echo "    For projects matching: {$qd->cooked_project_selector}\n";
-        echo "    release if: {$qd->release_criterion}\n";
+        echo "    release if: projects < {$qd->projects_target} or pages < {$qd->pages_target}\n";
```

Comment:
Could re-use the "convert queue criteria to display" here too.

Reply:
Done.